tests/tensorflow2/test_keras_mirrored.py

```diff
@@ -256,8 +256,10 @@ def test_save_all(out_dir, tf_eager_mode, workers):
     tr = create_trial_fast_refresh(out_dir)
     print(tr.tensor_names())
     if tf_eager_mode:
-        assert len(tr.tensor_names()) == (6 + 2 + 1 + 5 + 1 if is_tf_2_2() else 6 + 3 + 1 + 5 + 1)
-        # weights, metrics, losses, optimizer variables, scalar
+        assert len(tr.tensor_names()) == (
+            6 + 2 + 1 + 5 + 1 + 2 if is_tf_2_2() else 6 + 3 + 1 + 5 + 1 + 2
```

Comment:
this type of test 6+ 2 + 1 has no value, how do we know what is six tensors what is 2 tensors ? It seems cryptic and difficult to maintain.
We should have asserted tname.count == 6
and blah blah, rather than adding 6 + 2 + 1 ...